README.md

```diff
@@ -1,36 +1,98 @@
+
+`units` (string) Units to convert balance into; the default is "Wei".
+
+`displayFunc` (function(addr, bal, units)) If given, used to format and display the account data. 
 
 ### ContractData
 
+ContractData fetches data from a contract using the specified method and renders the result(s) as an unordered list (i.e., inside `<ul>` tags) by default. The default behaviour can be overwritten by specifying a `displayFunc` as decribed below; this enables the composition of more complex structures, like tables (see examples).
+
 `contract` (string, required) Name of the contract to call.
 
 `method` (string, required) Method of the contract to call.
 
 `methodArgs` (array) Arguments for the contract method call. EX: The address for an ERC20 balanceOf() function. The last argument can optionally be an options object with the typical form, `gas` and `gasPrice` keys.
 
-`hideIndicator` (boolean) If true, hides the loading indicator during contract state updates. Useful for things like ERC20 token symbols which do not change.
+`hideIndicator` (boolean) If true, hides the loading indicator during contract state updates. Useful for things like ERC20 token symbols that do not change.
+
+`displayFunc` (function(data)) If given, used to format and display the contract data.
 
 `toUtf8` (boolean) Converts the return value to a UTF-8 string before display.
 
 `toAscii` (boolean) Converts the return value to an Ascii string before display.
 
 ### ContractForm
 
+ContractForm creates a form for interacting with methods on smart contracts.
+
 `contract` (string, required) Name of the contract whose method will be the basis the form.
 
 `method` (string, required) Method whose inputs will be used to create corresponding form fields.
 
 `sendArgs` (object) An object specifying options for the transaction to be sent; namely: `from`, `gasPrice`, `gas` and `value`. Further explanataion of these parameters can be found [here in the web3 documentation](https://web3js.readthedocs.io/en/1.0/web3-eth-contract.html#id19).
 
-`labels` (array) Custom labels; will follow ABI input ordering. Useful for friendlier names. For example "_to" becoming "Recipient Address".
+`labels` (array) Custom labels; will follow ABI input ordering. Useful for friendlier names. EX: "to" becoming "Recipient Address".
+
+`hideMethod` (boolean) Hides the method name if true.
+
+## Example Usage
+
+### AccountData
+
+```<AccountData drizzle={drizzle} drizzleState={drizzleState} accountIndex='0' units='ether' displayFunc={(addr, bal, units) => (<div><strong>{addr}</strong>: {bal} {units}</div>)} />```
+
+> **0x...**: 3.43 ether
+
+
+### ContractData
+
+```
+<ContractData drizzle={drizzle} drizzleState={drizzleState} contract='MyContract' method='getStructKeys' />
+```
+
+produces:
+
+<ul><li>structKey1</li><li>structKey2</li><li>...</li></ul>
+
+```
+<ContractData drizzle={drizzle} drizzleState={drizzleState} contract='MyContract' method='getStruct' methodArgs=['1'] />
```

Comment:
The `methodArgs` prop is invalid syntax. It should be `methodArgs={['1']}` instead of `methodArgs=['1']`

Reply:
Fixed.